### ISSUE TITLE
cromwell: support relative file paths in input JSON

### ISSUE THREAD

**Issue by prihoda** [chanzuckerberg/miniwdl#230]
Currently, relative input paths are only supported when provided as command-line arguments, where they are translated to absolute paths. Providing relative input paths in the inputs JSON file passes them on unchanged and fails when trying to find them relative to the created cromwell folder:
```
Could not localize data/mockdata/seqs/First2000seqs_R1.fq -> /Users/.../20190830_101132_Workflow/cromwell/cromwell-executions/Workflow/ec53e465-8581-4027-bf54-f6c85e1deee9/call-Task/shard-1/inputs/46135967/First2000seqs_R1.fq:
	data/mockdata/seqs/First2000seqs_R1.fq doesn't exist
```

**Comment by mlin** [comment]
We'll need to walk all the (possibly nested) input values in the JSON, detect `File` values with relative paths, and prepend `../` or make them absolute

**Comment by mlin** [comment]
PS thanks for reporting @prihoda !

**Comment by mlin** [comment]
@prihoda should we interpret such paths relative to the directory in which the JSON file resides? or to the cwd?

**Comment by prihoda** [comment]
@mlin I would make it relative to the cwd, so that it is equivalent to supplying the file as command line argument. Cromwell also considers paths to be relative to cwd.

**Comment by mlin** [comment]
@prihoda This should be fixed up in v0.4.1. Thanks!
